app/views/tasks/show.html.slim

```diff
@@ -531,6 +531,13 @@
           div data-bs-toggle='tooltip' title=t('.guest.disabled.tooltip') data-bs-delay=150
             = button_tag class: 'btn btn-outline-dark dropdown-toggle disabled', 'data-bs-toggle': 'dropdown' do
               = t('.button.export')
+  - if policy(@task).generate_test?
```

Comment:
I can't see the image you attached. Could you please provide the code change you did of which you created this box?

Reply:
I could also remove that box completely and only include the one button at the bottom :)

Reply:
> I can't see the image you attached. 

The two images are attached in the collapsable "details" part, which is hidden by default. Clicking on the line with the black triangle reveals it.

<details>
<summary>Code Mockup</summary>

I was editing in the DOM viewer, thus I only hav the HTML. This is not necessary nice, just wanted to get the visual result (and I could have used a drawing tool, too).

```html
<div class="m-0 mb-3 p-0" style="
    height: 115px;
"><div class="border button-box h-100 m-0 mb-4 p-3" style="
    background-color: antiquewhite;
    height: 40px;
"><div class="button-description" style="
">Beta Feature: Generate unit tests for this Task.<br>As part of our research ...
</div><div class="wrapper"><div class="btn-group dropdown float-end" style="
    width: 200px;
"><button name="button" type="submit" class="btn btn-light" data-bs-toggle="dropdown" control-id="ControlID-1" style="
    /* width: 100px; */
">Generate Unit Tests</button><ul class="scrollable dropdown-menu" role="menu"><li><hr class="dropdown-divider"></li><li><form action="/collections" accept-charset="UTF-8" method="post"><input type="hidden" name="authenticity_token" value="lRcflGrerJYE57qJJCGOlhhznawS1lep2_QLgdGtrotZ7UnAIkgwhhNEfU-AKjcipmzBt--HYEthT65W-rfwIw" autocomplete="off"><div class="create-collection input-group"><input class="form-control custom-input" placeholder="Collection name" type="text" name="collection[title]" id="collection_title"><input value="359" autocomplete="off" type="hidden" name="collection[task_ids]" id="collection_task_ids"><input type="submit" name="commit" value="Create new Collection" class="btn btn-light submit" data-disable-with="Create Collection"></div></form></li></ul></div></div></div></div>
```

</details>

> I could also remove that box completely and only include the one button at the bottom :)

As you like.

Reply:
Yea I know. The first image shows. the second one is broken. But thanks

Reply:
Works on my machine 😅 

<img width="657" alt="Bildschirmfoto 2024-06-28 um 11 58 34" src="https://github.com/openHPI/codeharbor/assets/7300329/924c6c1e-423b-4cd8-aaa9-4be62a343a77">

<hr>


<img width="1904" alt="343725671-24eaf944-ddd9-4cee-8060-c326a98d357a" src="https://github.com/openHPI/codeharbor/assets/7300329/505648dc-ca96-4c4e-b513-35d2cd11d45c">

Reply:
They are all visible now :D thanks.

Reply:
Since you dropped the grey box right now (and we'll have the follow-up ticket in a bit), I would suggest to resolve this conversation here (please do so if you agree). Feel free to re-use some of the images in the follow-up ticket, if you like.